spot-ml/src/test/scala/org/apache/spot/proxy/ProxyWordCreationTest.scala

```diff
@@ -0,0 +1,287 @@
+package org.apache.spot.proxy
+
+
+import org.apache.log4j.{Level, LogManager}
+import org.apache.spot.SuspiciousConnectsArgumentParser.SuspiciousConnectsConfig
+import org.apache.spot.proxy.ProxySchema.Word
+import org.apache.spot.testutils.TestingSparkContextFlatSpec
+import org.scalatest.Matchers
+
+
+class ProxyWordCreationTest extends TestingSparkContextFlatSpec with Matchers {
+
+  val testConfigProxy = SuspiciousConnectsConfig(analysis = "proxy",
+    inputPath = "",
+    feedbackFile = "",
+    duplicationFactor = 1,
+    topicCount = 20,
+    hdfsScoredConnect = "",
+    threshold = 1.0d,
+    maxResults = 1000,
+    outputDelimiter = "\t",
+    ldaPRGSeed = None,
+    ldaMaxiterations = 20,
+    ldaAlpha = 1.02,
+    ldaBeta = 1.001)
+
+  "proxy word creation" should "return the correct word given the rules to form the word" in {
+
+    val logger = LogManager.getLogger("ProxyWordCreation")
+    logger.setLevel(Level.WARN)
+
+    // case class ProxyInput has the form:
+    // 1  p_date:String,
```

Comment:
Not aesthetically pleasant but it gives the user a more clear idea about how I'm building the unit tests and which fields (columns) have a direct impact on the final word.